src/i18n/langs/uk.js

```diff
@@ -0,0 +1,111 @@
+const uk = {
+  message: {
+    new_connection: 'Нове з`єднання',
+    refresh_connection: 'Оновити',
+    edit_connection: 'Редагувати з`єднання',
+    del_connection: 'Видалити з`єднання',
+    close_connection: 'Закрити з`єднання',
+    add_new_line: 'Додати новий рядок',
+    redis_version: 'Версія Redis',
+    process_id: 'ID процесу',
+    used_memory: 'Використано пам`яті',
+    used_memory_peak: 'Максимальний обсяг пам`яті',
+    used_memory_lua: 'Використання пам`яті Lua',
+    connected_clients: 'Підключено клієнтів',
+    total_connections_received: 'Кількість з`єднань',
+    total_commands_processed: 'Кількість команд',
+    key_statistics: 'Статистика по ключам',
+    all_redis_info: 'Вся інформація Redis',
+    server: 'Сервер',
+    memory: 'Оперативна пам`ять',
+    stats: 'Статистика',
+    settings: 'Налаштування',
+    confirm_to_delete_row_data: 'Підтвердити видалення даних рядка?',
+    delete_success: 'Успішно видалено',
+    delete_failed: 'Видалити не вдалося',
+    modify_success: 'Успішно змінено',
+    modify_failed: 'Зміни не вдалося',
+    add_success: 'Успішно додано',
+    add_failed: 'Додати не вдалося',
+    value_exists: 'Значення вже існує',
+    refresh_success: 'Успішно оновлено',
+    click_enter_to_rename: 'Натисніть сюди або Enter, щоб перейменувати ключ.',
+    click_enter_to_ttl: 'Натисніть сюди або Enter, щоб змінити час життя ключа TTL.',
+    confirm_to_delete_key: 'Підтвердіть видалення {key}?',
+    confirm_to_rename_key: 'Підтвердити перейменування {old} -> {new}?',
+    edit_line: 'Редагувати рядок',
+    auto_refresh: 'Автоматичне оновлення',
+    auto_refresh_tip: 'Автоматичне оновлення, оновлення кожні {interval} секунд',
+    key_not_exists: 'Ключ не існує',
+    collapse_all: 'Згорнути все',
+    expand_all: 'Розгорнути все',
+    json_format_failed: 'Не вдалося форматувати в JSON',
```

Comment:
lack of `msgpack_format_failed` just behind of `json_format_failed`, the master branch had added